python/tvm/relay/frontend/onnx.py

```diff
@@ -961,6 +961,18 @@ def _impl_v1(cls, inputs, attr, params):
         return _op.power(out, reci_p)
 
 
+class GlobalLpPool(OnnxOpConverter):
+    """Operator converter for GlobalLpPool."""
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        in_shape = infer_shape(inputs[0])
+        assert len(in_shape) >= 4, "input shape support NCHW or NCDHW"
+        attr["kernel_shape"] = in_shape[2:]
```

Comment:
Nit: I don't think this will work with dynamic shapes, but we shouldn't hit that often and  it's hard to support right now. Maybe add a Todo so we can keep an eye on it?